matrix/common/aws/batch_handler.py

```diff
@@ -11,42 +11,46 @@
 
 
 class BatchHandler:
-    def __init__(self, request_id, request_hash):
-        Logging.set_correlation_id(logger, value=request_hash)
-
-        self.request_id = request_id
-        self.request_hash = request_hash
-
+    def __init__(self):
         self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
-        self.s3_results_bucket = os.environ['S3_RESULTS_BUCKET']
+        self.s3_results_bucket = os.environ['MATRIX_RESULTS_BUCKET']
         self.job_queue_arn = os.environ['BATCH_CONVERTER_JOB_QUEUE_ARN']
         self.job_def_arn = os.environ['BATCH_CONVERTER_JOB_DEFINITION_ARN']
 
         self._client = boto3.client("batch", region_name=os.environ['AWS_DEFAULT_REGION'])
         self._cloudwatch_handler = CloudwatchHandler()
 
-    def schedule_matrix_conversion(self, format):
+    def schedule_matrix_conversion(self, request_id: str, format: str):
+        """
+        Schedule a matrix conversion job within aws batch infra
+
+        :param request_id: UUID identifying a matrix service request.
+        :param format: User requested output file format of final expression matrix.
+        """
+        Logging.set_correlation_id(logger, value=request_id)
         job_name = "-".join(["conversion",
                              self.deployment_stage,
-                             self.request_hash,
+                             request_id,
                              format])
 
         is_compressed = format == MatrixFormat.CSV.value or format == MatrixFormat.MTX.value
-        source_zarr_path = f"s3://{self.s3_results_bucket}/{self.request_hash}.zarr"
-        target_path = f"s3://{self.s3_results_bucket}/{self.request_hash}.{format}" + (".zip" if is_compressed else "")
+        source_expression_manifest = f"s3://{self.s3_results_bucket}/{request_id}/expressionmanifest"
+        source_cell_manifest = f"s3://{self.s3_results_bucket}/{request_id}/cell_metadatamanifest"
+        source_gene_manifest = f"s3://{self.s3_results_bucket}/{request_id}/gene_metadatamanifest"
+        target_path = f"s3://{self.s3_results_bucket}/{request_id}.{format}" + (".zip" if is_compressed else "")
```

Comment:
Do want to start to handle the user-requested name here?

Reply:
Yea, we'd probably want to pass that in from the api to driver and then store that user-requested-name in the output table. This can then be used in the batch handler when scheduling conversion job with target path. @mckinsel is there a ticket tracking this already?

Reply:
We decided against allowing a user requested filename in preference to that being managed by the data browser.